OpenQuestion/answer-question-1.md

```diff
@@ -0,0 +1,13 @@
+
+# Should we use tables in HTML? Why?
+
+## We should use tables in HTML incase
+Tabular Data Presentation: Use tables when you need to show data in a structured grid format with rows and columns.
+Comparison and Cross-Referencing: For instance, if you’re comparing product features, specifications, or prices, a table layout works well.
+Legacy Layouts and Compatibility:In older web development practices, tables were commonly used for page layout.
```

Comment:
Use unorder list for more readable
https://www.markdownguide.org/basic-syntax/#unordered-lists

Reply:
thanks, i added it

Reply:
@CBinh710 Still not resolved yet